modules/sdk-api/src/api.ts

```diff
@@ -176,7 +176,7 @@ export function setRequestQueryString(req: superagent.SuperAgentRequest): void {
  * Right now, it is very permissive with the timestamp variance.
  */
 export function verifyResponse(
-  bitgo: BitGo,
+  bitgo: BitGoAPI,
```

Comment:
Should this be `walletAPI`? We will have multiple services with different clients in the near future